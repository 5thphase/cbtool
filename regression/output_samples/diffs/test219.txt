 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_5 (xxx.xxx.xxx.xxx )...
