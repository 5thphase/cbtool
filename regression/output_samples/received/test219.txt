 status: Starting the attachment of vm_5, part of AI none...
 status: Waiting for vm_5 (cloud-assigned uuid 42AC253B-0EE0-51A5-B81D-B25F0EF46689), part of AI none, to start...
 status: Trying to establish network connectivity to vm_5 (cloud-assigned uuid 42AC253B-0EE0-51A5-B81D-B25F0EF46689), part of AI none, on IP address 230.45.146.138...
 status: Checking ssh accessibility on vm_5, part of AI none (ssh -p 22 klabuser@230.45.146.138)...
 status: Bootstrapping vm_5, part of AI none (creating file cb_os_paramaters.txt in "klabuser" user's home dir on 230.45.146.138)...
 status: Sending a copy of the code tree to vm_5, part of AI none (230.45.146.138)...
 status: Performing generic VM post_boot configuration on vm_5, part of AI none(230.45.146.138)...
VM object 0FC8B3F6-F86F-5839-8D91-B2D0086F8466 (named "vm_5") sucessfully attached to this experiment. It is ssh-accessible at the IP addresses 230.45.146.138 and 195.243.1.253 (42AC253B-0EE0-51A5-B81D-B25F0EF46689.simcloud.com).
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |90.17.211.77    |simhostd3                  |SUT           |private    |none    |none       |4BB20172-486A-5DCB-9DC1-A49A5B8AFE4A
|vm_2          |tinyvm              |pico32      |86.32.144.220   |simhostc12                 |LG            |private    |none    |none       |AAACFDFB-CC50-5D2E-9788-4ABE800E95B5
|vm_3          |tinyvm              |pico32      |71.248.153.186  |simhostb1                  |SUT           |private    |none    |none       |5CAE413F-749A-56CB-AE5D-83A2954057A4
|vm_4          |db2                 |copper32    |100.129.101.48  |simhostc12                 |SUT           |private    |none    |none       |9A02889F-F41B-5D9D-94AA-D6FCE06513DE
|vm_5          |netclient           |platinum64  |195.243.1.253   |simhostb11                 |SUT           |private    |none    |none       |0FC8B3F6-F86F-5839-8D91-B2D0086F8466

The VM object vm_5, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhostb11                           
|meta_tags                               |{'a': 'b', 'x': 'y', 'r': '2'}       
|size                                    |platinum64                           
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |5.49M                         
|Redis Version                                        |3.0.6                         
|Uptime (in seconds)                                  |8704445                       
|Total Connections Received                           |377219                        
|Total Commands Processed                             |1222739870                    
|Number of Keys                                       |592                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |5                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |5                             
|VMs ISSUED                                           |5                             
|VMs ARRIVED                                          |5                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |0                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |213                           

 ---------------- VMC-WIDE COUNTERS ----------------
|48E847B7-4EAD-595E-A603-912566454DDC (simzone_a) VM RESERVATIONS|2                             
|0F125A34-8350-57DE-AA00-635042C08FD4 (simzone_c) VM RESERVATIONS|1                             
|C1CCDA39-30E1-56FF-A6AE-36406AD2BA16 (simzone_d) VM RESERVATIONS|1                             
|67592E97-AC3C-50A4-B3D9-009931854DBD (simzone_b) VM RESERVATIONS|1                             

