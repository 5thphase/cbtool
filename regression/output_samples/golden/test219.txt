 status: Waiting for vm_5 (cloud-assigned uuid D9AB8189-3CA2-5A0F-8059-714F7D9B8F92) to start...
 status: Trying to establish network connectivity to vm_5 (cloud-assigned uuid D9AB8189-3CA2-5A0F-8059-714F7D9B8F92), on IP address 1.4.240.224...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_5 (1.4.240.224)...
 status: Bypassing generic VM post_boot configuration on vm_5 (1.4.240.224)...
VM object FF23191A-5599-58ED-9794-0B325BB697DA (named "vm_5") sucessfully attached to this experiment. It is ssh-accessible at the IP addresses 1.4.240.224 and 242.130.46.195 (D9AB8189-3CA2-5A0F-8059-714F7D9B8F92.simcloud.com).
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |4.81.141.120    |simhostb0                  |SUT           |private    |none    |none       |A04D80E7-0BB2-55AD-A873-E01F14DD1C2A
|vm_2          |tinyvm              |pico32      |238.26.35.71    |simhosta3                  |LG            |private    |none    |none       |846887FC-C541-5BFC-A523-DA3489AFC54F
|vm_3          |tinyvm              |pico32      |199.180.154.172 |simhostb1                  |SUT           |private    |none    |none       |608DB467-1061-5361-8C26-0FC544152663
|vm_4          |db2                 |copper32    |112.243.120.145 |simhostd14                 |SUT           |private    |none    |none       |492E5ED8-A975-5AE6-82B1-CBC1328B551B
|vm_5          |netclient           |platinum64  |242.130.46.195  |simhostb11                 |SUT           |private    |none    |none       |FF23191A-5599-58ED-9794-0B325BB697DA

The VM object vm_5, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhostb11                           
|meta_tags                               |{'a': 'b', 'x': 'y', 'r': '2'}       
|size                                    |platinum64                           
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |16.19M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12856079                      
|Total Connections Received                           |548503                        
|Total Commands Processed                             |42330733                      
|Number of Keys                                       |573                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |27                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |5                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |5                             
|VMs ISSUED                                           |5                             
|VMs ARRIVED                                          |5                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |0                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |213                           

 ---------------- VMC-WIDE COUNTERS ----------------
|A168E857-FDCA-5E02-98BB-32D5CA3934A5 (simzone_d) VM RESERVATIONS|1                             
|2A9C7A1E-D5EE-5CFE-BE56-2B4E298B71F7 (simzone_a) VM RESERVATIONS|1                             
|FB2A884A-73A4-5C91-B913-9AD4AF1FB28A (simzone_c) VM RESERVATIONS|1                             
|073C9CDF-01E7-5AA9-B92E-02C6DA6E75D6 (simzone_b) VM RESERVATIONS|2                             

